server/src/lib/model/db/import-locales.ts

```diff
@@ -258,6 +258,136 @@ const VARIANTS: Variant[] = [
+  },
+  {
+    locale_name: 'pwn',
+    variant_name: 'pinayuanan a kinaikacedasan (東排灣語)',
+    variant_token: 'pwn-eastern',
+  },
+  {
+    locale_name: 'pwn',
+    variant_name: 'pinayuanan a tjinuvirian (北排灣語)',
+    variant_token: 'pwn-northern',
+  },
+  {
+    locale_name: 'pwn',
+    variant_name: 'pinayuanan a tjinainavaljan (南排灣語)',
+    variant_token: 'pwn-southern',
+  },
+  {
+    locale_name: 'trv',
+    variant_name: 'Kari Seediq Tgdaya (德固達雅賽德克語)',
+    variant_token: 'trv-tgdaya',
+  },
+  {
+    locale_name: 'trv',
+    variant_name: 'Kari Sediq Toda (都達賽德克語)',
+    variant_token: 'trv-teuda',
+  },
+  {
+    locale_name: 'trv',
+    variant_name: 'Kari Seejiq Truku (德鹿谷賽德克語)',
+    variant_token: 'trv-truku',
+  },
+  {
+    locale_name: 'tay',
+    variant_name: 'keʼ na Squliq Tayal (賽考利克泰雅語)',
+    variant_token: 'tay-squliq',
+  },
+  {
+    locale_name: 'tay',
+    variant_name: 'kay na Sʼuli Tayan (澤敖利泰雅語)',
+    variant_token: 'tay-ciuli',
+  },
+  {
+    locale_name: 'tay',
+    variant_name: 'keʼ na Cʼoli Klesan Tayal (宜蘭澤敖利泰雅語)',
+    variant_token: 'tay-klesan',
+  },
+  {
+    locale_name: 'tay',
+    variant_name: 'ke na Cquliq Tayal (四季泰雅語)',
+    variant_token: 'tay-cquliq',
+  },
+  {
+    locale_name: 'tay',
+    variant_name: 'kayiʼ na Matuʼuwal (汶水泰雅語)',
+    variant_token: 'tay-matuuwal',
+  },
+  {
+    locale_name: 'tay',
+    variant_name: 'keʼ na Pingawan (萬大泰雅語)',
+    variant_token: 'tay-pingawan',
+  },
+  {
+    locale_name: 'dru',
+    variant_name: 'Ngodradrekai ka so Taromak (東魯凱語)',
+    variant_token: 'dru-taromak',
+  },
+  {
+    locale_name: 'dru',
+    variant_name: 'Ngudradreadrekadhane ka Veday (霧臺魯凱語)',
+    variant_token: 'dru-veday',
+  },
+  {
+    locale_name: 'dru',
+    variant_name: 'Ngudradreadrekadha ka Tailailabuwana (大武魯凱語)',
+    variant_token: 'dru-labuane',
+  },
+  {
+    locale_name: 'dru',
+    variant_name: 'ʼakoʼoponohae (萬山語)',
+    variant_token: 'dru-oponoho',
+  },
+  {
+    locale_name: 'dru',
+    variant_name: 'bakani Kongadavane (多納語)',
+    variant_token: 'dru-kunga',
+  },
+  {
+    locale_name: 'dru',
+    variant_name: 'teldredrekane (茂林語)',
+    variant_token: 'dru-teldreka',
+  },
+  {
+    locale_name: 'bnn',
+    variant_name: 'MalasTakitudu (卓群布農語)',
+    variant_token: 'bnn-takitudu',
+  },
+  {
+    locale_name: 'bnn',
+    variant_name: 'MalasTakibakha (卡群布農語)',
+    variant_token: 'bnn-bakha',
+  },
+  {
+    locale_name: 'bnn',
+    variant_name: 'MalasʼIsbubukun (郡群布農語)',
+    variant_token: 'bnn-bubukun',
+  },
+  {
+    locale_name: 'bnn',
+    variant_name: 'MalasTakbanuaz (巒群布農語)',
+    variant_token: 'bnn-banuaz',
+  },
+  {
+    locale_name: 'bnn',
+    variant_name: 'MalasTakivatan (丹群布農語)',
+    variant_token: 'bnn-vatan',
+  },
+  {
+    locale_name: 'nan-tw',
+    variant_name: '白話字 (POJ)',
+    variant_token: 'nan-tw-pehoeji',
+  },
+  {
+    locale_name: 'nan-tw',
+    variant_name: '台羅 (TL)',
+    variant_token: 'nan-tw-tailo',
```

Comment:
For non-cap `TW` in BCB codes, I'm doing it intentionally to follow how it appears in the repo (thus, how it could be in the current live system) and try to prevent any issue or cost of the engineer's time.

two examples of how TW  shows up differently in the source repo,

in library
<img width="1548" alt="Screen Shot 2024-12-03 01 27 14" src="https://github.com/user-attachments/assets/5e0d48f7-75cd-4d22-a750-f9944acc445a">

in DB migration
  
<img width="1494" alt="Screen Shot 2024-12-03 01 26 55" src="https://github.com/user-attachments/assets/0c7f3018-44a7-45fb-891d-edc3473c7298">

Reply:
Yes, this is wrong. They should be in caps. I don't believe we do any string processing on the variants (e.g. to extract any of the parts) at the moment, but we may do in the future, so it's best to future proof by following the standard. If necessary we can go back and fix (or retire) the ones that are invalid.

Reply:
fixed in #68ee468